### DISCUSSION TITLE
[SUGGESTION] - Break the python code apart

### DISCUSSION THREAD

**Discussion by altryne** [hlky/stable-diffusion-webui#208] [Feature suggestions]
If we extract the "ui" part out of the webui.py script, and then use it with an import, then we can do a few things : 

1) Develop UI faster - this would be possible as we could import the Gradio UI into a separate python file and run with debug=True (that's actually how I've been doing the dev, but it takes a long time to copy/paste back and forth) 

2) Write stubs for all functions we use to load the ui dev faster

3) Merge syncs and PR reviews will be easier

**Comment by hlky** [comment]
This definitely needs doing
